package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zenput/cordova-plugin-image-picker",
-  "version": "22.10.20-H17M24S13",
+  "version": "23.8.30-H08M20S22",
```

Comment:
Unlike Zencam, I don't think we have CI set up to deploy this automatically on merge to main. So, I think you'll have to manually run this from your local:

```sh
npm run calversion
npm publish
```

I would do that from this PR just before you merge to main instead of after, since it will make a diff in the code that needs to be committed. (This is from memory, so I may be slightly off. We should really document the publishing process in the README.)